provision/minikube/keycloak/templates/cockroach-operator/cockroach.yaml

```diff
@@ -0,0 +1,28 @@
+{{ if eq .Values.database "cockroach-operator" }}
+
+apiVersion: crdb.cockroachlabs.com/v1alpha1
+kind: CrdbCluster
+metadata:
+  # this translates to the name of the statefulset that is created
+  name: cockroach
+  namespace: {{ .Values.namespace }}
+spec:
+  dataStore:
+    pvc:
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: "20Gi"
+        volumeMode: Filesystem
+  tlsEnabled: true
+  # You can set either a version of the db or a specific image name
+  # cockroachDBVersion: v22.1.8
+  image:
+    name: cockroachdb/cockroach:v22.1.8
+  # nodes refers to the number of crdb pods that are created
+  # via the statefulset
+  nodes: 3
```

Comment:
@ahus1 do we want to mention this value of number of nodes that come up by default in the docs ?

Reply:
+1 on that. Do you think it should already be configurable as of today?

Reply:
TBH, I think we should expose scalability configuration outside of this ticket, be it Keycloak pods, CockroachDB pods etc., We can leave the configuration in the yaml definitions for now.

Reply:
pushed a commit to add the docs, after the discussion with @ahus1

Reply:
approved, thanks for updating the docs!